exporter/prometheusremotewriteexporter/README.md

```diff
@@ -50,6 +50,7 @@ The following settings can be optionally configured:
 - `headers`: additional headers attached to each HTTP request.
   - *Note the following headers cannot be changed: `Content-Encoding`, `Content-Type`, `X-Prometheus-Remote-Write-Version`, and `User-Agent`.*
 - `namespace`: prefix attached to each exported metric name.
+- `add_type_and_unit_suffixes`: If set to false, type and unit suffixes will not be added to metrics. Default: true.
```

Comment:
WDYT about `add_metric_suffixes` name? It's shorter and more consistent with the receiver's option. The docs/description can provide more details about what the suffixes are.

Reply:
makes sense.  Done.